gremlin-dotnet/src/Gremlin.Net/Structure/IO/GraphSON/TimeSpanDeserializer.cs

```diff
@@ -0,0 +1,37 @@
+﻿#region License
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#endregion
+
+using System.Xml;
+using Newtonsoft.Json.Linq;
+
+namespace Gremlin.Net.Structure.IO.GraphSON
+{
+    internal class TimeSpanDeserializer : IGraphSONDeserializer
```

Comment:
How about `DurationDeserializer` as a name for this class and `DurationSerializer` for the serializer?
I think that we always used the GraphSON type name for the names of the serializers and deserializers instead of the .NET type names (e.g., `UuidSerializer` instead of `GuidSerializer` or `DateSerializer` instead of `DateTimeOffsetSerializer`).